apps/frontend/src/app/Data/Characters/Furina/index.tsx

```diff
@@ -0,0 +1,689 @@
+    max_hp_: skillParam_gen.constellation2[4],
+  },
+  constellation4: {
+    energyRegen: skillParam_gen.constellation4[0],
+    cd: skillParam_gen.constellation4[1],
+  },
+  constellation6: {
+    duration: skillParam_gen.constellation6[0],
+    auto_dmgInc: skillParam_gen.constellation6[1],
+    ousiaCd: skillParam_gen.constellation6[2],
+    ousiaHeal: skillParam_gen.constellation6[3],
+    ousiaDuration: skillParam_gen.constellation6[4],
+    pneumaHpCost: skillParam_gen.constellation6[5],
+    pneumaAuto_dmgInc: skillParam_gen.constellation6[6],
+    triggers: skillParam_gen.constellation6[7],
+  },
+} as const
+
+const [condSkillHpConsumeStacksPath, condSkillHpConsumeStacks] = cond(
+  key,
+  'skillHpConsumeStacks'
+)
+const skillHpConsumeStacksArr = range(1, 4)
+const member_mult_ = lookup(
+  condSkillHpConsumeStacks,
+  objKeyMap(skillHpConsumeStacksArr, (stack) => percent(1 + 0.1 * stack)),
+  one
+)
+
+const [condBurstFanfarePath, condBurstFanfare] = cond(key, 'burstFanfare')
+const burstFanfareArr = range(50, dm.burst.maxFanfare, 50)
+const c1FanfareArr = range(
+  50,
+  dm.burst.maxFanfare + dm.constellation1.bonusFanfare,
+  50
+)
+const clampedFanfareNum = lookup(
+  condBurstFanfare,
+  objKeyMap(c1FanfareArr, (stacks) =>
+    stacks > 450 ? greaterEq(input.constellation, 1, stacks) : constant(stacks)
+  ),
+  naught
+)
+const burstFanfare_all_dmg_ = prod(
+  clampedFanfareNum,
+  subscript(input.total.burstIndex, dm.burst.dmgIncRatio, {
+    unit: '%',
+    fixed: 2,
+  })
+)
+const burstFanfare_incHeal_ = prod(
+  clampedFanfareNum,
+  subscript(input.total.burstIndex, dm.burst.heal_ratio, {
+    unit: '%',
+    fixed: 2,
+  })
+)
+
+const a4Member_dmg_ = greaterEq(
+  input.asc,
+  4,
+  min(
+    prod(percent(dm.passive2.member_dmg_), input.total.hp, percent(1 / 1000)),
+    percent(dm.passive2.max_member_dmg_)
+  )
+)
+const a4HealInterval = greaterEq(
+  input.asc,
+  4,
+  max(
+    prod(
+      percent(-dm.passive2.interval_dec_),
+      input.total.hp,
+      percent(1 / 1000)
+    ),
+    percent(-dm.passive2.max_interval_dec_)
+  )
+)
+const member_addl: Data = {
+  premod: {
+    skill_dmg_: a4Member_dmg_,
+  },
+}
+
+const [condC2OverstackPath, condC2Overstack] = cond(key, 'c2Overstack')
+const c2OverstackArr = range(50, dm.constellation2.stacks, 50)
+const c2Overstack_hp_ = greaterEq(
+  input.constellation,
+  2,
+  lookup(
+    condC2Overstack,
+    objKeyMap(c2OverstackArr, (stack) =>
+      prod(stack, percent(dm.constellation2.hp_, { fixed: 2 }))
+    ),
+    naught
+  )
+)
+
+const [condc6Path, condc6] = cond(key, 'c6')
+const [condC6PneumaPath, condC6Pneuma] = cond(key, 'c6Pneuma')
+
+const c6_infusion = greaterEqStr(
+  input.constellation,
+  6,
+  equalStr(condc6, 'on', 'hydro')
+)
+const c6_auto_dmgInc = greaterEq(
+  input.constellation,
+  6,
+  equal(
+    condc6,
+    'on',
+    prod(
+      sum(
+        percent(dm.constellation6.auto_dmgInc),
```

Comment:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/22147385-a922-4b49-bd22-d318468f1028)
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/ac9bc508-51b0-4e51-8cbf-5bd4ed5025d0)
This looks clunky imo. You cannot tell what has changed after clicking the checkbox. Why change it from the original?

Reply:
I tried to make the conditionals stack under the same header, but failed.